scripts/globals/abyssea.lua

```diff
@@ -399,3 +415,170 @@ function abysseaOnEventFinish(player,csid,option)
         return true
     end
 end
+
+local abyssea_zones =
+{
+    15, -- ABYSSEA_KONSCHTAT
+    45, -- ABYSSEA_TAHRONGI
+    132,-- ABYSSEA_LA_THEINE
+    215,-- ABYSSEA_ATTOHWA
+    216,-- ABYSSEA_MISAREAUX
+    217,-- ABYSSEA_VUNKERL
+    218,-- ABYSSEA_ALTEPA
+    253,-- ABYSSEA_ULEGUERAND
+    254 -- ABYSSEA_GRAUBERG
+    --255 -- ABYSSEA_EMPYREAL_PARADOX
+}
+
+function isInAbysseaZone(player)
+    for index, zone in ipairs(abyssea_zones) do
+        if zone == player:getZoneID() then
+            return true
+        end
+    end
+    return false
+end
```

Comment:
Can this just `return player:getCurrentRegion() == tpz.region.ABYSSEA`?

Reply:
Seconded.

Reply:
good call sir's

Reply:
done